src/DragSelect.vue

```diff
@@ -7,17 +7,36 @@
 </template>
 
 <script>
+  // Expands array to have a unique function that filters out non-unique values
+  // eslint-disable-next-line no-extend-native
+  Array.prototype.unique = function () {
```

Comment:
Instead of extending the Array prototype, let's create a function instead.

Reply:
ok, will look into this.

Reply:
Moved it to a method outside of the Vue object called uniqueArray and call this. Could/Should be moved to common section if this library would ever grow beyong the initial component